### PR TITLE
Feature/translate cards

### PR DESCRIPTION
**Pull request by JamirLeal**
### What does this PR do?

- This PR translates the content of the sponsor packages cards, also the content is displayed according to the user selected language. Put a props of language in cards working according to the parent component Contact Sponsors to set the cards information. 

#### Related issues

#15

#### Merge Checklist:

- [x] The branch is up-to-date (i.e. rebased) with the base branch
- [ ] No code style issues

### CODE REVIEW

File: src/components/Contact/ContactSponsorUs/ContactSponsorPackages/ContactSponsorPackages.js [hunk 1]
```diff
@@ -1,16 +1,32 @@
...
+      language: props.language
+    };
+  }
+
+  /* 
+  * Component needs to update every time we change the props in the parent component (button to change idiom)
+  * by this we set a new state every time this happens.
+  */
+  componentDidUpdate( prevProps ) {
+    if(prevProps.language !== this.props.language) {
```
Comment (RicardoChapaRomero) [inline src/components/Contact/ContactSponsorUs/ContactSponsorPackages/ContactSponsorPackages.js:27]:
```suggestion
    if (prevProps.language !== this.props.language) {
```

File: src/components/Contact/ContactSponsorUs/ContactSponsorUs.js [hunk 2]
```diff
@@ -21,29 +21,45 @@ class ContactSponsorUs extends Component {
...
 
   sponsorUsCallback() {
     window.open(this.sponsorsData.url_contact, '_blank');
   }
 
+  /**
+   * Do the change of the state parameters when the language changes (language, translate button label, and title).
+   */
   translate() {
     if(this.state.language === 'en') {
```
Comment (RicardoChapaRomero) [inline src/components/Contact/ContactSponsorUs/ContactSponsorUs.js:38]:
```suggestion
    if (this.state.language === 'en') {
```

**Review by RicardoChapaRomero — APPROVED** [review]
LGTM! I addressed some nit comments but is good to merge

**Comment by JamirLeal** [comment]
Changes done!

**Pull request merged**